app/src/pages/settings/ModelsTable.tsx

```diff
@@ -229,13 +217,98 @@ export function ModelsTable({
         accessorKey: "namePattern",
         cell: TextCell,
       },
-      makeCostColumn("input", "input cost"),
-      makeCostColumn("output", "output cost"),
-      makeCostColumn("cacheRead", "cache read cost"),
-      makeCostColumn("cacheWrite", "cache write cost"),
-      makeCostColumn("promptAudio", "prompt audio cost"),
-      makeCostColumn("completionAudio", "completion audio cost"),
-      makeCostColumn("reasoning", "reasoning cost"),
+      {
+        header: "prompt cost",
+        id: "input",
+        accessorFn: (row) => getRowCost(row, "input"),
+        meta: { textAlign: "right" },
+        sortingFn: sortCostColumnFn,
+        sortUndefined: "last",
+        cell: ({ row }) => {
+          const inputCost = getRowCost(row.original, "input");
+          return (
+            <Flex justifyContent="end" alignItems="center" gap="size-100">
+              <span>{inputCost}</span>
+              <TooltipTrigger delay={0}>
+                <IconButton size="S" aria-label="Input cost details">
+                  <Icon svg={<Icons.MoreHorizontalOutline />} />
+                </IconButton>
```

Comment:
This looks like it should do something on click. For what it is worth, you can use tooltips on non-buttons by wrapping the trigger (text, icon, whatever) with `<Focusable>`

Reply:
I'm not 100% sure from an a11y perspective things that have tooltips don't have to be buttons of some kind. Even with focusable I think you need the "role=button" which basically makes it so you have a button. So I'm not entirely convinced this pattern isn't just the right thing. 

Let's see what the emergent pattern is. It might make sense to just componentize this pattern.